avm/res/batch/batch-account/README.md

```diff
@@ -1106,6 +1106,17 @@ Array of role assignments to create.
 
 - Required: No
 - Type: array
+- Roles configurable by name:
```

Comment:
Can we try running the script on a resource having roleassignemnts on child resources as well (e.g., key vault)? I guess no difference  unless udt are implemented for the child resource? Is that the case?

Reply:
Right, there is no difference. I just ran it for Kvlt and uploaded the changes for you to review

Reply:
Thanks @AlexanderSehr, so if we e.g. added UDT for kv key, which has rbac at child level, then we would see the kv key roles added also at kv/parent level, similarly to what we see for cross referenced modules. Is that correct?

Reply:
ping @eriqua @eriqua

Reply:
@MRMcake